CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## Changelog
 
+### v1.2.1 May 29, 2019
+
+* Update to be compatible with Android Reader SDK 1.3.0. (No delay capture)
```

Comment:
"(no delay capture)"  is not clear.  Reader SDK 1.30 added delay capture.   Is this note saying that the plugin was upgraded but excluded the 1.3.0 delay capture feature?